src/tutorial/scheduling/integer-cycling.rst

```diff
@@ -362,46 +362,33 @@ Recurrence Sections
            # Repeat every third cycle.
            P3 = baz
 
-.. digraph:: example
+.. image:: ../../img/reccurrance-sections.svg
    :align: center
 
-   subgraph cluster_1 {
-       label = 1
-       style = dashed
-       "foo.1" [label="foo\n1"]
-       "bar.1" [label="bar\n1"]
-       "baz.1" [label="baz\n1"]
-   }
-
-   subgraph cluster_2 {
-       label = 2
-       style = dashed
-       "foo.2" [label="foo\n2"]
-   }
-
-   subgraph cluster_3 {
-       label = 3
-       style = dashed
-       "foo.3" [label="foo\n3"]
-       "bar.3" [label="bar\n3"]
-   }
-
 .. nextslide::
 
 .. ifnotslides::
 
-   By default recurrences start at the :term:`initial cycle point`, however it
-   is possible to make them start at an arbitrary cycle point. This is done by
-   writing the cycle point and the recurrence separated by a forward slash
-   (``/``), e.g. ``5/P3`` means repeat every third cycle starting *from* cycle
-   number 5.
+   You may have noticed that in the previous example that it is not possible to
+   get even numbered cycles. By default recurrences start at the
+   :term:`initial cycle point`, however it is possible to make them start at an
+   arbitrary cycle point. This is done by writing the cycle point and the
+   recurrence separated by a forward slash (``/``), e.g. ``5/P3`` means repeat
+   every third cycle starting *from* cycle number 5. Therefore, if you wanted
+   a graph to occur every even cycle point you would use ``2/P2``.
 
    The start point of a recurrence can also be defined as an offset from the
    :term:`initial cycle point`, e.g. ``+P5/P3`` means repeat every third cycle
    starting 5 cycles *after* the initial cycle point.
 
 .. ifslides::
 
+   ``2/P2``
+      Repeat every even cycle (If your initial cycle point was odd)
+
+   .. image:: ../../img/reccurrance-sections2.svg
+      :align: center
+
    ``5/P3``
       Repeat every third cycle starting *from* cycle number 5.
    ``+P5/P3``
```

Comment:
Should we make this `+P2/P2`?

Reply:
Also should we update the datetime example to something along the lines of:

`2000/P2Y`
and
`+P2Y/P2y`
?

Reply:
Possibly...

Reply:
> Should we make this `+P2/P2`?

I think it is good to demo that it's not only every even cycle. Odd and even numbers feel like a special case (after all there's a special word for them) and I could imagine users thinking that we **only** support **that** alternative recurrence.

Reply:
> Also should we update the datetime example to something along the lines of:
> 
> `2000/P2Y`
> and
> `+P2Y/P2y`
> ?

I made it every fourth year for one example and left the other as one. For the reason outlined above I think that more variety is better here.